### ISSUE TITLE
Allow full script configuration

### ISSUE THREAD

**Issue by no1melman** [caseyWebb/generator-npm-init#53]
Obviously you have enabled the test property of scripts to be configured, would be cool if you could configure the entire block, something like:

```
const npmOptions = {
    'skip-name': true,
    'skip-description': true,
    'skip-version': true,
    'skip-main': true,
    'skip-scripts': true,
    'skip-repo': true,
    'skip-keywords': true,
    'skip-author': true,
    'skip-license': true,

    name,
    version: '1.0.0',
    main: 'index.js',
    author,
    license: 'ISC',
    scripts: {
        "start": "webpack-dev-server --progress --watch",
        "build:old": "./node_modules/.bin/babel index.js --out-file ./dist/main.js",
        "build": "node ./tools/build.js",
        "test": "echo \"Error: no test specified\" && exit 1"
    }
};
```

I can create a PR if you want?

**Comment by caseyWebb** [comment]
Submit a pr, I'll merge it

**Comment by no1melman** [comment]
@caseyWebb Done :)

**Comment by caseyWebb** [comment]
Published in v1.4.0
